src/sass/card/card.scss

```diff
@@ -0,0 +1,135 @@
+@import "../mixins/private/bubble-mixins";
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: $card-size-default;
+}
+
+.card__container {
+    position: relative;
+    height: $card-size-default;
+}
+
+@media screen and (max-width: $_screen-size-MD) {
+    .card {
+        width: $card-size-large;
+
+        .card__container {
+            height: $card-size-large;
+        }
+    }
+}
```

Comment:
We usually put our media queries at the end of the file, so they're easy to be found.